README.md

```diff
@@ -25,4 +25,4 @@ To run a solution's tests using a Docker container, follow these steps:
    1. Once the script has completed, the test results will be written to `<output-directory>/results.json`.
 
 
-[test-runner-interface]: https://github.com/exercism/automated-tests/blob/master/docs/interface.md
\ No newline at end of file
+[test-runner-interface]: https://github.com/exercism/docs/blob/main/building/tooling/test-runners/interface.md
```

Comment:
Copying the comment directly in a review:

> Please could this link to the website: https://exercism.org/docs/building/tooling/test-runners/interface
>
>The docs repo isn't stable in terms of URLs and isn't intended to be directly linked to.

Reply:
so in general - all markdown links should point to their "deployed" web counterparts?

Reply:
I think that is a "wherever possible" thing. Not all markdown files inside repos have deployed counterparts on the webpage.

Reply:
Moreso the docs don't actually allow traversing links on GitHub, because their paths aren't correct for GitHub.